docs/UserGuide.md

```diff
@@ -277,33 +288,88 @@ Format: `unprioritze INDEX`
+an example with steps to follow:
 
-### Marking assignments as not done : `undone`
+1) To mark assignments with the name "" and "" shown in the figure below as done, you can simply enter `done 2 3done 2 3` into
+the command line based on the indexes circled and labelled in the figure.
+
+DIAGRAM
+
+2) The two assignments will be marked as done and each assignment will have a green tick displayed as circled in the figure. 
+
+DIAGRAM
+
+3) A message that includes the information of the assignments marked as done will be displayed in the Command Box.
+
+DIAGRAM
 
-Marks the specified assignment as not done.
+<div markdown="block" class="alert alert-primary">
+ **:clipboard: Pointers to note:**<br>
+* At least one index must be present. For example, `done` without any index will not work.
+* The indexes **must be a positive integer** 1, 2, 3, …​
+* The indexes **must be found in your assignment list**
+* The indexes **must not be duplicated**. For example, `done 4 4` will not work.
+</div>
+
+
+### Marking assignments as not done : `undone`
 
 Format: `undone INDEX`
 
-**Tip:**
-Assignments are marked as not done by default.
+You can unmark assignments that you have previously marked as done by specifying the assignment `INDEX` as shown in your
+ list.
 
-* Marks the assignment at the specified `INDEX` as not done.
-* The index refers to the index number shown in the displayed assignment list.
+You can unmark `one` done assignment at a time. Here is an example with steps to follow:
+
+1) If you have accidentally marked the assignment with the name " " as done,
+you can simply enter `undone X` into the command line based on the index circled in the figure.
+
+DIAGRAM
+
+2) The green tick associated to the assignment is no longer displayed as shown in the diagram below.
+
+DIAGRAM
+
+3) A message that includes the information of your assignment marked as undone will be displayed in the Command Box.
+
+DIAGRAM 
+
+<div markdown="block" class="alert alert-primary">
+ **:clipboard: Pointers to note:**<br>
+* Assignments are marked as not done**by default**
+* The index must be present. For example, `undone` without any index will not work.
 * The index **must be a positive integer** 1, 2, 3, …​
+* The index **must be found in your assignment list**
+</div>
 
-### Exiting the program : `exit`
+### Clearing all assignments: `clear`
 
-Exits the program.
+Format: `clear`
+
+You can clear all your existing assignments and lessons at the start of every semester.
+
+<div markdown="span" class="alert alert-success">
+**:bulb: Tip:**
+If you cleared all your assignments and lessons by accident, use `undo` to retrieve your deleted data.
+</div>
+
+### Accessing help: `help`
+
+Format: `help`
+
+You can see a summary of all the commands that you can use in ProductiveNUS.
+
+### Exiting the program : `exit`
 
 Format: `exit`
 
+Exits the program.
+
```

Comment:
Minor nitpick but maybe phrasing it as "You can use this command to exit the program" would be more consistent with the rest of the UG in terms of the style of writing :)

Reply:
changed!